### ISSUE TITLE
Exists the possibility of write log but without show in console?

### ISSUE THREAD

**Issue by debelop13** [megahertz/electron-log#81]
Hi,

I have two questions:

It can save log in log.log but without showing in console?

Or I can save in different logs (log.log) the renderer and the main?

Thanks !!

**Comment by megahertz** [comment]
Hi.
1. Sure, just disable console transport. See README.md
2. This feature will be available in version 3.

**Comment by debelop13** [comment]
In reference to point 1, I want this in only some log, not all.

Can be?

Thanks!!

**Comment by megahertz** [comment]
The only way is to use helper function like this:
```js
function logSilent(...args) {
  const oldLevel = log.transports.console.level;
  log.transports.console.level = false;
  log.info(...args);
  log.transports.console.level = oldLevel;
}
```

**Comment by megahertz** [comment]
Part 2  is implemented in v3.0.0-beta2. Part 1 can now be implemented using hooks.
